lib/core/test/unit/Cardano/Pool/DB/Arbitrary.hs

```diff
@@ -251,7 +255,7 @@ instance Arbitrary SinglePoolCertificateSequence where
             -- We must start with a registration certificate:
             certificates <- (:)
                 <$> (Registration <$> arbitrary)
-                <*> arbitrary
+                <*> scale (min 10) arbitrary
```

Comment:
Maybe add a `limitScale = scale . min` if this is going to be used a lot.
But I'm not sure it's good to overuse this.
Normally you would want all sub-values of an arbitrary value to grow in proportion to the scale value.

Reply:
>  But I'm not sure it's good to overuse this.

I agree. Although it's mainly use for database tests at this stage which kinda makes sense to me still. In quite many database property tests we've been generating huge lists of values which doesn't really bring anything more than slowness. 

Perhaps we could simply use a smaller scale factor instead of clamping the whole thing, a slightly modified version of `log` for instance or `sqrt`.

Reply:
see: 80b0f8d8b79d6c2d56175d775a651ac2acc2217c